### PR TITLE
Modules packages

### PR DESCRIPTION
**Pull request by mnupura**
Modules created - Module_Calculator.py, Program_Calculator.py
Package created - Program_Package.ipynb, My_Package folder

### CODE REVIEW

File: My_Package/Subtract_Pack/Subtract_Pack.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def subtract(num1,num2):
+	print("I am in Subtract package.")
+	result=num1-num2
```
Comment (jaipurandare) [inline My_Package/Subtract_Pack/Subtract_Pack.py:3]: This trial comment

**Review by jaipurandare — CHANGES_REQUESTED** [review]
Remove .pyc files

File: My_Package/Subtract_Pack/Subtract_Pack.py (hunk 1 above)
Comment (mnupura) [reply My_Package/Subtract_Pack/Subtract_Pack.py:3]: .pyc removed and ignored using .gitignore file

**Review by jaipurandare — APPROVED** [review]
LGTM

**Pull request merged**